### PR TITLE
new  story

### PR DESCRIPTION
**Pull request by gchhetri12**
New update. Please review and merge.

### CODE REVIEW

File: story1.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+intial story
```
Comment (gchhetri12) [inline story1.txt:1]: this is an initial story. got it.

File: story1.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+initial commit
```
Comment (gchhetri12) [inline story1.txt:2]: inital comit. got it.

**Review by gchhetri12 — COMMENTED** [review]
accepted.

**Review by chhetri97 — COMMENTED** [review]
accepted.

**Pull request merged**